### PR TITLE
Add links to README and for reporting bugs

### PR DESCRIPTION
**Pull request by arielshaqed**
Also bump version number so NPMJS will update their page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,12 @@
 {
   "name": "@reshuffle/db-admin",
-  "version": "0.0.10",
+  "version": "0.0.11",
   "description": "Super Simple Admin interface for Reshuffle DB",
   "main": "dbAdmin.js",
+  "homepage": "https://github.com/reshufflehq/reshuffle-db-admin#readme",
+  "bugs": {
+    "url": "https://github.com/reshufflehq/reshuffle-db-admin/issues"
+  }
```
Comment (arielshaqed) [inline package.json]:
```suggestion
  },
```
(thinking of getting my `{"I"  :" ❤️ JSON"  }` tattoo).

File: package.json [hunk 2]
```diff
@@ -1,8 +1,12 @@
 {
   "name": "@reshuffle/db-admin",
-  "version": "0.0.10",
+  "version": "0.0.11",
```
Comment (arielshaqed) [inline package.json:3]: Version bump is "only" to update the page on npmjs.com, right?

**Review by arielshaqed — COMMENTED** [review]
Don't think current version will pass `npm`.

**Review by zolga — APPROVED** [review]
LGTM

**Comment by arielshaqed** [comment]
Thanks!
Pulling, and will publish a new version.

**Comment by arielshaqed** [comment]
```bash
flutterby:reshuffle-db-admin ariels$ npm publish
npm notice 
npm notice 📦  @reshuffle/db-admin@0.0.11
npm notice === Tarball Contents === 
npm notice 516B  package.json
npm notice 5.4kB dbAdmin.js  
npm notice 316B  ReadMe.md   
npm notice === Tarball Details === 
npm notice name:          @reshuffle/db-admin                     
npm notice version:       0.0.11                                  
npm notice package size:  2.1 kB                                  
npm notice unpacked size: 6.3 kB                                  
npm notice shasum:        27532d5f5cc9f3c2f9a2313e61e073ec3f48d65d
npm notice integrity:     sha512-dhOSH8MDWb82H[...]5kr+FeFzfwHpQ==
npm notice total files:   3                                       
npm notice 
+ @reshuffle/db-admin@0.0.11
```

**Pull request merged**